src/config/ConfigLog.ts

```diff
@@ -11,19 +11,44 @@ import {
   LoggerFactoryOptions,
   LogGroupRule,
   LogLevel,
+  getLogControl,
+  LogGroupControlSettings,
 } from 'typescript-logging'
 
-// Create options instance and specify 2 LogGroupRules:
-// * One for any logger with a name starting with model, to log on debug
-// * The second one for anything else to log on info
+// Create options instance and specify 1 LogGroupRule:
+// * LogLevel Error on default, env DEBUG = 'true' changes Level to Debug.
 const options = new LoggerFactoryOptions().addLogGroupRule(
-  new LogGroupRule(new RegExp('.+'), LogLevel.Debug)
+  new LogGroupRule(
+    new RegExp('.+'),
+    process.env.DEBUG && process.env.DEBUG === 'true'
+      ? LogLevel.Debug
+      : LogLevel.Error
+  )
 )
-
 // Create a named loggerfactory and pass in the options and export the factory.
 // Named is since version 0.2.+ (it's recommended for future usage)
 // eslint-disable-next-line import/prefer-default-export
 export const factory = LFService.createNamedLoggerFactory(
   'LoggerFactory',
   options
 )
+
+/**
+ *  Changes all existing Loggers of our default Factory with id 0 to the intended Level.
+ *
+ * @param level The intended LogLevel.
```

Comment:
Could potentially add that this ranges from 0 to 5 but do as you wish.

Reply:
Yes good addition.

I wish though the LogLevel enum had this kind of information at runtime.